androidshared/src/main/res/values/material_theme_polyfill.xml

```diff
@@ -9,9 +9,9 @@
 <resources>
 
     <style name="Theme.AndroidShared.MaterialComponents.Polyfill.TextInputLayout" parent="">
```

Comment:
Using these polyfill overlays instead of changing the attributes on the field should make these easy to remove when we switch to proper primary and secondary colors: we can just delete the overlays and remove them wherever they are used.